NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -108,15 +108,17 @@ public enum ServerEventType : int
         /// Executed when blood is placed.
         /// </summary>
 		/// <remarks>
-		/// Parameters: <see cref="IPlayer"/> player, <see cref="Vector3"/> position.
+		/// Parameters: <see cref="IPlayer"/> bleeder, <see cref="IPlayer"/> target, <see cref="Vector3"/> position.
```

Comment:
I don't know whether to call it a "target" or "shooter" since I don't remember if SCPs can bleed players.

Reply:
As I said commented in the old PR - you were right. There is no way to get the shooter.

Reply:
Fixed